### DISCUSSION TITLE
What java version use for develop?

### DISCUSSION THREAD

**Discussion by VladShyrokyi** [hdescottes/GdxGame#51] [Q&A] [answered]
I am clone project by idea. I has installed java 17, but ide suggest what  gradle 7.1.1 not support it java version.

```
Found invalid Gradle JVM configuration
JDK 17.0.3 isn't compatible with Gradle 7.1.1. Please fix JAVA_HOME environment variable.
```

**Comment by hdescottes** [comment, marked as answer]
Hello,
I developped this project under Java 11 :)
According to the [matrix compatibility](https://docs.gradle.org/current/userguide/compatibility.html), you might want to upgrade the gradle version to 7.3 to be compatible with java 17.

**Reply by VladShyrokyi** [reply]
Thanks)
